packages/ocap-kernel/src/Kernel.ts

```diff
@@ -587,6 +587,7 @@ export class Kernel {
    * Gracefully stop the kernel without deleting vats.
    */
   async stop(): Promise<void> {
+    await this.#remoteManager.stopRemoteComms();
```

Comment:
I like having a provision for an orderly shutdown, but I fear that this will rarely happen in the wild. Instead, I think it far more likely that somebody just quits their browser, taking any running kernel and its vats with it. Outside parties that had a communications relationship with whatever had been running in there might have to wait a very long time before being able to reconnect. I'm not sure what this means in terms of maximum retry backoff limits, but I suspect it means *something*.